app/views/spree/static_content/show.html.erb

```diff
@@ -2,11 +2,13 @@
   <%= render :partial => @page.layout %>
 <% else %>
   <% content_for :sidebar do %>
-    <% if @products && @taxon %>
-      <%= render :partial => "spree/shared/filters" %>
-    <% elsif @taxonomies %>
-      <%= render :partial => "spree/shared/taxonomies" %>
-    <% end %>
+    <div data-hook="homepage_sidebar_navigation">
+      <% if "spree/products" == params[:controller] && @taxon %>
```

Comment:
Wondering if this is actually required? Static pages are rendered by StaticContentController, so when would `params[:controller]` be equal to `spree/product` here?

Reply:
It's just for parity with solidus_frontend:

https://github.com/solidusio/solidus/blob/dd42c4a10edd99ed629f5e52641c59961da0ae91/frontend/app/views/spree/products/index.html.erb#L1-L9

I agree we'll need to have a better solution for this in the future tho.